package.json

```diff
@@ -1,51 +1,21 @@
 {
-  "name": "reaction-api",
-  "version": "4.2.0",
+  "name": "reaction",
+  "version": "1.0.0",
   "description": "Reaction is a modern reactive, real-time event driven ecommerce platform.",
-  "main": "./src/index.js",
-  "type": "module",
-  "engines": {
-    "node": ">=14.18.1"
-  },
-  "release": {
-    "branches": [
-      "trunk"
-    ],
-    "plugins": [
-      "@semantic-release/commit-analyzer",
-      "@semantic-release/release-notes-generator",
-      [
-        "@semantic-release/exec",
-        {
-          "prepareCmd": "sed -i 's/${lastRelease.version}/${nextRelease.version}/g' docker-compose.yml"
-        }
-      ],
-      [
-        "@semantic-release/changelog",
-        {
-          "changelogFile": "CHANGELOG.md"
-        }
-      ],
-      [
-        "@semantic-release/npm",
-        {
-          "npmPublish": false
-        }
-      ],
-      "@semantic-release/github",
-      [
-        "@semantic-release/git",
-        {
-          "assets": [
-            "package.json",
-            "package-lock.json",
-            "CHANGELOG.md",
-            "docker-compose.yml"
-          ],
-          "message": "chore(release): v${nextRelease.version} [skip ci]"
-        }
-      ]
-    ]
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "npm run start:dev -w=apps/reaction",
+    "start:meteor-blaze-app": "npm run start -w=apps/meteor-blaze-app",
+    "build:packages": "npm run build -w=packages/file-collections -w=packages/file-collections-sa-base -w=packages/file-collections-sa-gridfs -w=packages/logger -w=packages/random",
+    "test": "npm run test --if-present --ws",
+    "test:integration:query": "npm run test:integration:query --workspace=apps/reaction",
+    "test:integration:mutation": "npm run test:integration:mutation --workspace=apps/reaction",
+    "changeset": "changeset",
+    "release": "changeset publish",
+    "lint": "eslint . --resolve-plugins-relative-to .",
+    "lint:docker": "npx --quiet --package @reactioncommerce/ci-scripts@1.6.2 lint-dockerfiles --workspace=apps/reaction",
+    "lint:gql": "npm run lint:gql -w=apps/reaction -w=packages/api-core -w=packages/api-plugin-surcharges",
```

Comment:
I think it would be better to do `npm run lint:gql --if-present --ws` so that it will lint GraphQL for all packages that have it set up.